sourcegraph/sourcegraph
# Add events from frictionless cody to allowlist

The events need to make it into Amplitude to determine if our efforts are successful. 

## Test plan

* Ensure megration is applied in the DB
* Step through the home page to ensure events are triggered and are added to the allowlist
* Click the signup to get access button on Sorucegraph.com/search and fire an event
* Observe event in Amplitude
